docs/api.rst

```diff
@@ -8927,17 +8927,27 @@ Adding EVM accounts to all EVM chains
 
       {
         "result": {
-            "eth": [
-                "0x78b0AD50E768D2376C6BA7de33F426ecE4e03e0B",
-                "0x19b0AD50E768D2376C6BA7de32F426ecE4e03e0b"
-            ],
-            "avax": ["0x78b0AD50E768D2376C6BA7de33F426ecE4e03e0B"],
-            "optimism": ["0x78b0AD50E768D2376C6BA7de33F426ecE4e03e0B"]
-        },
+            "success": {
+               "eth": [
+                  "0x78b0AD50E768D2376C6BA7de33F426ecE4e03e0B",
+                  "0x19b0AD50E768D2376C6BA7de32F426ecE4e03e0b"
+               ],
+               "avax": ["0x78b0AD50E768D2376C6BA7de33F426ecE4e03e0B"],
+               "optimism": ["0x78b0AD50E768D2376C6BA7de33F426ecE4e03e0B"]
+            },
+            "already_have_been_added": {
+               "gnosis": ["0x19b0AD50E768D2376C6BA7de32F426ecE4e03e0b"]
+            },
+            "failed": {
+               "avax": ["0x19b0AD50E768D2376C6BA7de32F426ecE4e03e0b"]
+            },
+         },
         "message": ""
       }
 
-   :resjson list result: A mapping containing the evm chain keys and which addresses were added for each chain.
+   :resjson list success: A mapping containing the evm chain keys and which addresses were added for each chain.
+   :resjson list already_have_been_added: A mapping containing the evm chain keys and which addresses weren't added because they were added before.
+   :resjson list failed: A mapping containing the evm chain keys and which addresses failed to get added for each chain.
```

Comment:
Just a suggestion for the names of the keys to be a bit more succint

```suggestion
   :resjson list added: A mapping containing the evm chain keys and which addresses were added for each chain.
   :resjson list existed: A mapping containing the evm chain keys and which addresses weren't added because they were added before.
   :resjson list failed: A mapping containing the evm chain keys and which addresses failed to get added for each chain.
```